src/Symfony/Component/Console/Helper/HelperSet.php

```diff
@@ -89,7 +91,7 @@ public function getCommand()
     }
 
     /**
-     * @return Helper[]
+     * @return \Traversable<Helper>
```

Comment:
I am not sure whether we should use `\Traversable<Helper>` or `\Traversable<Helper>|Helper[]`. Technically the first is correct but I believe that the latter is a common workaround for IDEs not being able to make sense of the generic `\Traversable`. Though I'm not sure this applies to the `getIterator` method in case the method is not called but the object is iterated over in a `foreach`.